WooCommerce/Resources/Assets.xcassets/LaunchLogo.imageset/Contents.json

```diff
@@ -2,16 +2,7 @@
   "images" : [
     {
       "idiom" : "universal",
-      "scale" : "1x"
-    },
-    {
-      "idiom" : "universal",
-      "filename" : "WooLaunchLogo@2x.png",
-      "scale" : "2x"
-    },
-    {
-      "idiom" : "universal",
-      "scale" : "3x"
+      "filename" : "woo-icon-120x120-1.pdf"
```

Comment:
super nit and non-blocking: I feel like the "-1" in the filename implies that there's another file "woo-icon-120x120.pdf", and we probably don't need the dimension in the filename since it's a vector format :)

Reply:
not sure if it makes any visual difference, I usually turn on the "Preserve Vector Data" option for a PDF image:

<img width="679" alt="Screen Shot 2020-02-12 at 8 46 21 PM" src="https://user-images.githubusercontent.com/1945542/74336231-da5ca200-4dd8-11ea-85d4-09942b8adf09.png">

based on the [Apple doc](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_ref-Asset_Catalog_Format/ImageSetType.html):

> preserves-vector-representation | Boolean | Set to true to preserve the vector information for a PDF file.